### ISSUE TITLE
cc-header-addon / cc-header-app: provide visible label for ids ?

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#562]
The `appId` / `realAppId` input fields have no visible labels.
![image](https://user-images.githubusercontent.com/100240294/189312467-da460d7c-4cb4-4500-aef8-9828eed4a4fa.png)

Adding visible labels for these fields makes the header less compact and do not provide much info.
In this specific case, I would argue it's not that big of a deal and that we could let it go.

These elements are readonly and they are only meant to provide a content that you can easily copy.
This means the typical issue with the lack of visible labels (you don't know what is expected of you) is irrelevant here.

Still, users that use voice navigation would need a visible label to know what to say in order to reach these fields easily so it's not that easy.

This is really debatable, i'm not 100% comfortable with this to be honest. I'd say this is very low prio anyway, we provided visually hidden labels in the meantime so they can be reached by assistive technologies.
These visually hidden labels match the terminology used by the support team and the clever tools (app id, real app id, addon id, real addon id).
